superset-frontend/src/explore/actions/saveModalActions.js

```diff
@@ -215,3 +218,23 @@ export const getDashboard = dashboardId => async dispatch => {
     throw error;
   }
 };
+
+//  Get dashboards the slice is added to
+export const getSliceDashboards = slice => async dispatch => {
+  if (slice) {
+    try {
+      const response = await SupersetClient.get({
+        endpoint: `/api/v1/chart/${slice.slice_id}?q=${rison.encode({
+          columns: ['dashboards.id'],
+        })}`,
+      });
+
+      return response.json.result.dashboards.map(({ id }) => id);
+    } catch (error) {
+      dispatch(saveSliceFailed());
+      throw error;
+    }
+  }
+
+  return [];
```

Comment:
`return []` should be removed if remove the `if clause`

Reply:
I don't want to remove the `if` clause; returning `[]` isn't exception behavior (see above).